servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/PublisherFlatMapMerge.java

```diff
@@ -383,6 +393,7 @@ private void drainPending() {
                                     ++emittedCount;
                                 }
                             } catch (Throwable cause) {
+                                ++emittedCount;
```

Comment:
Why this change is required?

Reply:
if an exception is caught here it is bcz an item has been delivered and the subscriber threw, so we need decrement the count (missed previously).